tools/run_tests/xds_k8s_test_driver/requirements.lock

```diff
@@ -0,0 +1,40 @@
+absl-py==0.15.0
```

Comment:
Is there a way for us to enforce requirements.txt being in sync with requirements.lock? So that if people add a module, or update the version of an existing one, they don't forget to update the lock file?

Reply:
I think the PR does this already. If someone adds a module to only `requirements.txt` and not `requirements.lock`, that module will not appear in the tests and will therefore fail CI.

You could come up with a more sophisticated mechanism, like IfChange, but that seems like it would be quite a bit more effort for diminishing marginal returns.